lib/hanami-router.rb

```diff
@@ -1 +1,2 @@
 require 'hanami/router'
+require 'hanami/http_router_monkey_patch'
```

Comment:
This is something that concerns `Hanami::Routing::HttpRouter` which is the engine of `Hanami::Router`. This require should be moved in `lib/hanami/routing/http_router.rb`.